graphql_compiler/compiler/metadata.py

```diff
@@ -161,6 +172,24 @@ def get_location_info(self, location):
                                  u'{}'.format(location))
         return location_info
 
+    @property
+    def outputs(self):
```

Comment:
can you rename it to `get_all_outputs` for consistency with other getters here, and add a docstring explaining that it returns an iterable of tuples?

Reply:
This is meant to be a *property* consistent with `tags` and `registered_locations`. Not sure if we have a preference between properties and get_all. Maybe in a future PR?